mne_bids/read.py

```diff
@@ -502,6 +504,22 @@ def _handle_channels_reading(channels_fname, raw):
     return raw
 
 
+@requires_freesurfer('mri_convert')
+def _make_minimal_subject_dir(t1_bids_path):
+    """Make a minimal freesurfer subject directory for recovering trans."""
+    subjects_dir = mne.utils._TempDir()
+    subject = t1_bids_path.subject
+    env = os.environ.copy()
+    subjects_dir = get_subjects_dir(subjects_dir, raise_error=True)
+    env['SUBJECTS_DIR'] = subjects_dir
+    if not op.isdir(op.join(subjects_dir, subject, 'mri')):
+        os.makedirs(op.join(subjects_dir, subject, 'mri'))
```

Comment:
I think using `pathlib.Path` here, the code gets much cleaner:
```suggestion
    mri_dir = Path(subjects_dir) / subject / 'mri'
    mri_dir.mkdir(parents=True, exist_ok=True)
```